frontend/src/PostContentEditor.jsx

```diff
@@ -6,7 +6,7 @@ import React from "react";
 import { Editor } from "react-draft-wysiwyg";
 import { EditorState, convertToRaw } from 'draft-js';
 import draftToMarkdown from 'draftjs-to-markdown';
-import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
+import "../node_modules/react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
```

Comment:
We should avoid importing directly from node_modules - this should work instead

import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';